### ISSUE TITLE
REST interface to access tournament information

### ISSUE THREAD

**Issue by larswolter** [ultical/ultical#64]
Implementing a REST interface to access the tournament information would allow other applications in use by teams to reference those tournaments without managing their own lists out of sync. 

This rest interface should only include the tournament infos (at least name, date and probably location) not the roster information.

An alternative would be an RSS-feed of upcoming tournaments, but this would bee less effective in cases of syncing and retrieval of past tournaments.

**Comment by bslbckr** [comment]
Yeah, I really like this idea and the REST Interface already exists. In my
opinion, we solely have to remove access restrictions on the respective
resources.

Are you interested in the other registered teams as well?

Am 25.07.2016 2:38 nachm. schrieb "Lars Wolter" notifications@github.com:

**Comment by larswolter** [comment]
As i wrote, the main interest would be the tournament infos themself, but of course, every info that can be shown in addition to the tournament can be usefull to users.

**Comment by bastrapp** [comment]
That would be another great use case of #47 and after its resolve easy to implement

**Comment by stm2** [comment]
Hey, this would be great for Ultiorganizer as well. I'm tired of hacking in all the info every time. I'd need tournament info and registered team info, not rosters, I guess.

**Comment by JensFriebe** [comment]
Links to API:

https://www.dfv-turniere.de/api/tournaments
https://www.dfv-turniere.de/api/events
https://www.dfv-turniere.de/api/events/basics

https://api.ultical.com/v3/event
https://api.ultical.com/v3/federation
https://api.ultical.com/v3/country
